src/index.js

```diff
@@ -0,0 +1,180 @@
+
+const state = {
+    city: 'Seattle',
+    temp: 72,
+    cityNameTypeBox: null,
+};
+
+const loadInput = () => {
+    state.cityNameTypeBox.document.getElementById('cityNameTypeBox');
+};
+
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+};
+
+
+const findLatAndLong = () => {
+    let latitude, longitude;
+
+    axios
+        .get('http://127.0.0.1:5000/location', {
+            params: {
+            q: `${state.cityNameTypeBox}`,
+            }
+        })
```

Comment:
`state.cityNameTypeBox` doesn't ever get a value, currently, so this always seems to query for the location `null`. Which....does *something*, but I'm not sure what!

You can get the city name in your query by changing lines 8-25 like this:

```javascript
const loadInput = () => {
    state.cityNameTypeBox = document.getElementById('cityNameTypeBox').value;
};

const convertKtoF = (temp) => {
    return (temp - 273.15) * (9 / 5) + 32;
};


const findLatAndLong = () => {
    let latitude, longitude;
    loadInput();

    axios
        .get('http://127.0.0.1:5000/location', {
            params: {
            q: `${state.cityNameTypeBox}`,
            }
        })
```

This puts the value of the `cityNameTypeBox` input field into `state.cityNameTypeBox`, then calls `loadInput` within `findLatAndLong` so that it's usable by the time you call your axios get.